lib/combobox.js

```diff
@@ -238,18 +238,6 @@ module.exports = React.createClass({
     this.refs.input.select();
   },
 
-  inputKeydownMap: {
-    8: 'removeLastToken',
-    9: 'selectOnEnter',
-    13: 'selectOnEnter',
-    27: 'hideOnEscape',
-    32: 'selectOnEnter',
-    38: 'focusPrevious',
-    40: 'focusNext',
-    186: 'selectOnEnter',
-    188: 'selectOnEnter'
-  },
```

Comment:
I think this change is too destructive and will cause merge conflicts when trying to merge future versions of the original repo into our repo. 

An option here is to leave this object defined in the class and update the `handleKeydown` function to use this map and merge it with the keymap being passed as prop. This keeps existing functionality the same and through our prop we can override existing values in the map or just append new mappings.

For example the decoraid implementation would look something like:
`<TokenInput inputKeydownMap={{189: 'selectOnEnter'}} />`
which would result in a map that looks something like:
```
{
8: 'removeLastToken',
9: 'selectOnEnter',
13: 'selectOnEnter',
27: 'hideOnEscape',
32: 'selectOnEnter',
38: 'focusPrevious',
40: 'focusNext',
186: 'selectOnEnter',
188: 'selectOnEnter',
189: 'selectOnEnter'
}
```
This approach would prevent the consumer of the component from needing to know all the original keymappings just to provide a new keymapping.